src/renderer/component/shapeShift/internal/active-shift.jsx

```diff
@@ -94,7 +94,7 @@ class ActiveShapeShift extends React.PureComponent<Props> {
             {shiftDepositAddress && (
               <FormRow verticallyCentered padded>
                 <QRCode value={shiftDepositAddress} paddingRight />
-                <Address address={shiftDepositAddress} showCopyButton padded />
+                <CopyableText copyable={shiftDepositAddress} showCopyButton padded />
```

Comment:
Just realized you are calling this prop `copyable`. A better name would be `value`.